packages/web-ui/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,182 @@
+      '--badge-border-color': 'var(--badge-outline-border-color-inverted)',
+    },
+  },
+  [COLORSCHEME_SELECTORS.cyan]: {
+    '--badge-soft-color': colors.cyan900,
+    '--badge-soft-background-color': colors.cyan200,
+    '--badge-strong-color': colors.cyan50,
+    '--badge-strong-background-color': colors.cyan600,
+    '--badge-outline-color': colors.cyan900,
+    '--badge-outline-border-color': colors.cyan600,
+    '--badge-outline-color-inverted': colors.cyan50,
+    '--badge-outline-border-color-inverted': colors.cyan500,
+  },
+  [COLORSCHEME_SELECTORS.green]: {
+    '--badge-soft-color': colors.green900,
+    '--badge-soft-background-color': colors.green200,
+    '--badge-strong-color': colors.green50,
+    '--badge-strong-background-color': colors.green600,
+    '--badge-outline-color': colors.green900,
+    '--badge-outline-border-color': colors.green600,
+    '--badge-outline-color-inverted': colors.green50,
+    '--badge-outline-border-color-inverted': colors.green400,
+  },
+  [COLORSCHEME_SELECTORS.red]: {
+    '--badge-soft-color': colors.red900,
+    '--badge-soft-background-color': colors.red200,
+    '--badge-strong-color': colors.red50,
+    '--badge-strong-background-color': colors.red500,
+    '--badge-outline-color': colors.red900,
+    '--badge-outline-border-color': colors.red500,
+    '--badge-outline-color-inverted': colors.red50,
+    '--badge-outline-border-color-inverted': colors.red500,
+  },
+  [COLORSCHEME_SELECTORS.gold]: {
+    '--badge-soft-color': colors.gold900,
+    '--badge-soft-background-color': colors.gold200,
+    '--badge-strong-color': colors.gold900,
+    '--badge-strong-background-color': colors.gold300,
+    '--badge-outline-color': colors.gold900,
+    '--badge-outline-border-color': colors.gold500,
+    '--badge-outline-color-inverted': colors.gold50,
+    '--badge-outline-border-color-inverted': colors.gold400,
+  },
+  [COLORSCHEME_SELECTORS.grey]: {
+    '--badge-soft-color': colors.grey900,
+    '--badge-soft-background-color': colors.grey100,
+    '--badge-strong-color': colors.grey900,
+    '--badge-strong-background-color': colors.grey200,
+    '--badge-outline-color': colors.grey900,
+    '--badge-outline-border-color': colors.grey500,
+    '--badge-outline-color-inverted': colors.grey50,
+    '--badge-outline-border-color-inverted': colors.grey300,
+  },
+});
+
+/**
+ * Provide additional context (such as status), or use Badge to draw attention to another interface element.
+ *
+ * > This component does not need to be wrapped in a `ThemeProvider` and can be used standalone with other component libraries.
+ */
+export const Badge = React.forwardRef<
+  React.ElementRef<'span'>,
+  React.PropsWithChildren<PropsWithSx<BadgeProps>>
+>(
+  (
+    {
+      className,
+      colorScheme = 'cyan',
+      variant = 'soft',
+      bottomRadiusZero,
+      compact = false,
+      inverted = false,
+      ...props
+    },
+    ref
+  ) => {
+    const { isBrandBackground } = useBackground();
```

Comment:
Not sure that the use of `inverted` is always true on a `BrandBackground`, we could have the badge on other backgrounds (i.e a darker shade of cyan or grey). And,  if the prefix _brand_ is scalable in the future as these colours are not brand only 🤔 and they are subject to change

Reply:
Currently, it is always true on a brand background (midnight & purple), and there is the `inverted` prop if needed on other backgrounds. If necessary we can always update the name in the future.

Reply:
> Currently, it is always true on a brand background (midnight & purple)

This would be hard to validate on production code because people has been using a local component to emulate Badge.

My only suggestion is to clarify on the documents that although midnight & purple are the most common usages, it is not restrictive to those bgs.